Blockzilla/UserAgent.swift

```diff
@@ -52,6 +52,12 @@ class UserAgent {
 
         return webViewUserAgent
     }
+    
+    open static func getDesktopUserAgent() -> String {
+        // TODO: check if this is suffficient. Chose this user agent instead of Firefox's method as Firefox fails to load desktop on several sites (i.e. Facebook)
```

Comment:
Cool! If this is a better method, file this in Bugzilla against Firefox iOS so it will get fixed.